branching_xblock/static/js/src/branching_xblock.js

```diff
@@ -24,11 +41,22 @@ function BranchingXBlock(runtime, element) {
         );
 
         // Hint
-        const $hint = $el.find('[data-role="hint"]');
-        if (state.enable_hints && node.hint) {
-          $hint.html(`<strong>Hint:</strong> ${node.hint}`).show();
+        const nodeId = node && node.id ? node.id : null;
```

Comment:
```suggestion
        const nodeId = node?.id || null;
```
A bit easier to read.